notification-bbpress.php

```diff
@@ -5,73 +5,83 @@
  * Plugin URI: https://wordpress.org/plugins/notification-bbpress/
  * Author: BracketSpace
  * Author URI: https://bracketspace.com
- * Version: 3.0.1
+ * Version: 4.0.0
```

Comment:
```suggestion
 * Version: [Next]
```

Reply:
Dobra, ogarnąłem, że tutaj to w ogóle się nie robi zmian, bo to się automatycznie regexem zmienia xd